app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -18,37 +18,26 @@ inset_text:
     text: |-
       <p><h2>End of the international relocation payment (IRP) pilot</h2></p>
       <br>
-      <p>The 2-year IRP pilot which ran from 1 September 2023 to 31 May 2025 has ended and is not being extended.<p/>
+      <p>The 2-year IRP pilot which ran from 1 September 2023 to 31 May 2025 has ended and is not being extended.</p> 
       <br>
-      <p>No new applications will be accepted for those starting a qualifying teaching role in England from 1 June 2025.</p> 
-      <br>
-      <p>If you have started a qualifying teaching role between 1 March 2024 and 31 May 2025, you must apply for your first instalment of IRP before the window closes on 30 June 2025.</p>
-      <br>
-      <p>Teachers who have already received their first instalment will still be able to apply for their second instalment from September 2025.</p>
-      <br>
-      <p>Before you apply, you must read the guidance on this page to check you are eligible for the IRP.</p>
-      <br>
-      <p><a href="https://www.claim-additional-teaching-payment.service.gov.uk/get-a-teacher-relocation-payment/landing-page">Apply for the international relocation payment</a>.</p>
+      <p>No new applications will be accepted for teachers:
+      <p><ul><li>starting a qualifying teaching role in England from 1 June 2025</li></p>
+      <p><li>who started a qualifying teaching role between 1 March 2024 and 31 May 2025</li></p></ul>
```

Comment:
missing an opening `<ul>`

...all alternatively, get rid of the wrapping paragraphs